marketplaceItems/avatarCustomization101/appResources/appData/avatarCustomization101.js

```diff
@@ -0,0 +1,759 @@
+//
+//  avatarCustomization101.js
+//
+//  Blueprint App to teach users how to use materials, blendshapes, and flow for their avatars.
+// 
+//  Created by Robin Wilson and Mark Brosche 2/20/2019
+//  Avatar created by Jimi Youm 2/20/2019
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+/* global GlobalDebugger */
+
+(function () {
+    // Modules
+    Script.include(Script.resolvePath("./resources/modules/flow.js?v12"));
+
+    var AppUi = Script.require("appUi"),
+        URL = Script.resolvePath("./resources/avatarCustomization101_ui.html?v12344555"),
+        CONFIG = Script.require(Script.resolvePath("./resources/config.js?v22222")),
+        BLENDSHAPE_DATA = Script.require(Script.resolvePath("./resources/modules/blendshapes.js?v1")),
+        MATERIAL_DATA = Script.require(Script.resolvePath("./resources/modules/materials.js?v123")),
+        AVATAR_URL = Script.resolvePath("./resources/avatar/avatar.fst");
+
+    var DEBUG = true;
```

Comment:
Remember to make this false before merging!

Reply:
On it!